message_test.go

```diff
@@ -19,7 +20,7 @@ func TestMessage(t *testing.T) {
 		Labels:    map[string]string{"source": "test"},
 		Message:   "Hello world!",
 	}
-	if m.String() != "2006-01-02T15:04:05Z\terror\tsource=test\tHello world!" {
+	if !strings.Contains(m.String(), "2006-01-02T15:04:05Z\terror\tsource=test\tHello world!") {
 		t.Fatalf("Incorrect message string: %s", m.String())
 	}
```

Comment:
Now that we have the terminal device conditionalization, consider reverting this change.  (And, possibly the one in `writer_golog_test.go` as well.)